kernel/mman.c

```diff
@@ -638,11 +638,14 @@ static int _munmap(myst_mman_t* mman, void* addr, size_t length)
     // perform scrub efficiently. The prot_vector based implementation has
     // complication of inconsistent prot within a VAD. Skip scrubbing for
     // now.
-#if 0    
```

Comment:
Maybe you removed this by mistake?

Reply:
It was broken before and I kind of figured the current solution was better. I can add both checks though.